src/order/rel_iso.lean

```diff
@@ -337,6 +337,12 @@ end
 @[simp] theorem of_monotone_coe [is_trichotomous α r] [is_asymm β s] (f : α → β) (H) :
   (@of_monotone _ _ r s _ _ f H : α → β) = f := rfl
 
+/-- An order embedding from an empty type. -/
+def of_is_empty (r : α → α → Prop) (s : β → β → Prop) [is_empty α] : r ↪r s :=
+{ to_fun := is_empty_elim,
+  inj' := is_empty_elim,
+  map_rel_iff' := is_empty_elim }
```

Comment:
```suggestion
⟨embedding.of_is_empty, is_empty_elim⟩
```

Reply:
This way you get a better definitional equality for `to_embedding`. Otherwise LGTM.
bors d+

Reply:
I'm glad that `embedding.of_is_empty` exists and matches the naming convention I went for here.